bvt/op-fvt-basic.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- IBM_PROLOG_BEGIN_TAG                                                   -->
+<!-- This is an automatically generated prolog.                             -->
+<!--                                                                        -->
+<!-- $Source: op-auto-test/bvt/op-fvt-basic.xml $                            -->
```

Comment:
Is the though that we'd put other OP tests in here or just OPAL?  If just OPAL then we should put opal in the file name.  Similar comment with other files in this review that have generic file names but OPAL specific testing.